src/wp-includes/rest-api/endpoints/class-wp-rest-autosaves-controller.php

```diff
@@ -360,35 +360,34 @@ public function create_post_autosave( $post_data ) {
 			return $post;
 		}
 
+		// Only create an autosave when the it is different than the saved post.
```

Comment:
```suggestion
		// Only create an autosave when it is different than the saved post.
```

Reply:
thanks for the correction! (think "to" should still be "than" though)

Reply:
> The one big difference between these terms is that 'different from' and 'different to' are standard in British English. 'Different than', meanwhile, is primarily used in American English.

That explains it! Happy to close this one as resolved, thanks @adamsilverstein!

Reply:
Oh, I had no idea the British English usage was different. Good to know, thanks! 

I'll accept your suggestion to fix the extraneous "the", then revert the language change bit to keep the American usage which is I believe standard for core.